### PR TITLE
PR List: replace special characters with string which can be parsed

### PR DESCRIPTION
**Pull request by ojacques**
when issuing `gh pr --list`, I could not easily parse PR test status because of special characters that were used. I understand this is probably an issue with my terminal.
Yet, with this PR, I can programmatically parse the status, looking for `OK` / `NOK` strings. 

Before:
image

With the change:
image

It still looks nice thanks to the coloring.

### CODE REVIEW

**Comment by protoEvangelion** [comment]
@ojacques Thanks for opening this PR looking into this now :+1:

**Comment by zenorocha** [comment]
@protoEvangelion this is a good alternative: https://github.com/sindresorhus/figures

**Comment by protoEvangelion** [comment]
@zenorocha Thanks for the tip!

@ojacques can you test these unicode alternatives?

**checkmark**: \u2714
**xmark**: \u2718

If they work you can add those to this PR and I will merge.

By the way, are you on Windows? If the above doesn't work we can try the figures library :wink:

**Comment by ojacques** [comment]
@protoEvangelion - I tried with the unicodes you indicated, same result.
I can try to add `figures` as a dependency and see how it goes. 

In my opinion, because `gh` is a command line, using unicodes chars to display info is not ideal. Those can display wrong, and are hard to parse.

**Comment by protoEvangelion** [comment]
@ojacques Thanks for checking.Please let me know if `figures` works and I would be happy to merge that :+1:

**Comment by protoEvangelion** [comment]
@ojacques Any update on this?

**Comment by ojacques** [comment]
@protoEvangelion - I am not on Windows, yet I had trouble parsing the output.
Now, I will close this as I was finally able to parse the unicode output.

Thank you!
